RaveBusinessLogic/BRAT.xml

```diff
@@ -11,11 +11,6 @@
           <Node xpathlabel="Name" xpath="HTMLFile" type="report" />
           <Node label="Capacity">
             <Children>
-              <Node label="Existing Dam Capacity" xpath="Vector" type="line" symbology="oCC_EX" id="ex_capacity" />
-              <Node label="Existing Dam Size" xpath="Vector" type="line" symbology="mCC_EX_CT" />
-              <Node label="Historic Dam Capacity" xpath="Vector" type="line" symbology="oCC_HPE" id="his_capacity" />
-              <Node label="Historic Dam Size" xpath="Vector" type="line" symbology="mCC_HPE_CT" />

```

Comment:
Why are we getting rid of historic dam size?

Reply:
@joewheaton, @MattReimer, and I went through and got rid of the lines that had 'Vector' xpaths, and left all of the geopackage xpaths, but all of the capacity outputs are still there, including historic dam size. Does this answer your question?

Reply:
Yeah. They were all just duplicates that would have caused all kinds of havoc in QRAVE.